### PR TITLE
Clean up NLS files from kernel projects

### PR DESCRIPTION
**Pull request by jhanders34**
- Remove translated files from boot.cmdline, core and nested since we only ship the ones from boot.  The english one doesn't even exist any longer in cmdline, core and nested. Update bnd to say globalize: false to make sure if they get added again they don't get put in the bundle.
- Add CWWKE0017E back into the boot one.  It should have been added back when we consolidated the english since core, cmdline and nested all share it.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#libby #build

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -100,7 +100,9 @@ error.secPermission=CWWKE0016E: The bootstrap JAR file requires the AllPermissio
 error.secPermission.explanation=The Java 2 Security permissions are insufficient for the bootstrap JAR file to launch the runtime environment and the OSGi framework.
 error.secPermission.useraction=Ensure that the bootstrap JAR file has been granted AllPermission privileges in the Java 2 Security policy file.
 
-# CWWKE0017E removed because it's only used in com.ibm.ws.kernel.boot.cmdline. Do not reuse.
+error.badBitmode=CWWKE0017E: The process detected a 31-bit JVM which is not supported. The runtime environment requires 64-bit Java 8 or higher.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]:
```suggestion
error.badBitmode=CWWKE0017E: The process detected a 31-bit JVM that is not supported. The runtime environment requires 64-bit Java 8 or higher.
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -100,7 +100,9 @@ error.secPermission=CWWKE0016E: The bootstrap JAR file requires the AllPermissio
...
+error.badBitmode.explanation=The JAVA_HOME system variable is set to point to a version of Java that is 31-bit.
+error.badBitmode.useraction=Adjust the JAVA_HOME system variable to point to 64-bit Java 8 or higher.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]:
```suggestion
error.badBitmode.useraction=Adjust the JAVA_HOME system variable to point to the 64-bit Java 8 version or higher.
```

**Review by ramkumar-k-9286 — APPROVED** [review]

**Review by hlhoots — APPROVED** [review]
LGTM

**Comment by jhanders34** [comment]
#libby

**Comment by jhanders34** [comment]
#libby

**Pull request merged**